.husky/pre-commit

```diff
@@ -0,0 +1 @@
+npx lint-staged
```

Comment:
(Optional)
Add this on top to make sure it runs correctly across different environments:
`
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"
`

Reply:
Done